src/components/autocomplete-grid/autocomplete-grid-button/readme.md

```diff
@@ -0,0 +1,129 @@
+inside a [sbb-autocomplete-grid](/docs/components-sbb-autocomplete-grid-sbb-autocomplete-grid--docs).
+
+```html
+<sbb-form-field label="Label">
+  <input />
+  <sbb-autocomplete-grid>
+    <sbb-autocomplete-grid-row>
+      <sbb-autocomplete-grid-option value="1">Option 1</sbb-autocomplete-grid-option>
+      <sbb-autocomplete-grid-cell>
+        <sbb-autocomplete-grid-button icon-name="pen-small"></sbb-autocomplete-grid-button>
+      </sbb-autocomplete-grid-cell>
+    </sbb-autocomplete-grid-row>
+    <sbb-autocomplete-grid-row>
+      <sbb-autocomplete-grid-option value="2">Option 2</sbb-autocomplete-grid-option>
+      <sbb-autocomplete-grid-cell>
+        <sbb-autocomplete-grid-button icon-name="trash-small"></sbb-autocomplete-grid-button>
+      </sbb-autocomplete-grid-cell>
+    </sbb-autocomplete-grid-row>
+  </sbb-autocomplete-grid>
+</sbb-form-field>
+```
+
+## Slots
+
+The component can display a `sbb-icon` using the `iconName` property or via custom content using the `icon` slot.
+
+```html
+<sbb-autocomplete-grid-button icon-name="pen-small"></sbb-autocomplete-grid-button>
+
+<sbb-autocomplete-grid-button>
+  <sbb-icon slot="icon" name="info"></sbb-icon>
+</sbb-autocomplete-grid-button>
+```
+
+## Button properties
+
+The component is internally rendered as a button,
+accepting its associated properties (`type`, `name`, `value` and `form`).
+
+```html
+<sbb-autocomplete-grid-button
+  icon-name="coins-small"
+  type="button"
+  name="tickets"
+  form="buy"
+  value="tickets"
+>
+</sbb-autocomplete-grid-button>
+```
+
+## Style
+
+The component has a negative variant which can be set using the `negative` property.
+
+The component can be displayed in `disabled` state using the self-named property.
+
+```html
+<sbb-autocomplete-grid-button icon-name="pen-small" negative></sbb-autocomplete-grid-button>
+
+<sbb-autocomplete-grid-button icon-name="pen-small" disabled></sbb-autocomplete-grid-button>
+```
+
+If the component is used within a [sbb-autocomplete-grid-optgroup](/docs/components-sbb-autocomplete-grid-sbb-autocomplete-grid-optgroup--docs),
+it can be disabled by disabling the optgroup.
+
+## Interactions
+
+When the button is clicked, an event is triggered; the behavior is up to the consumer.
+It's possible to fetch the button's related `sbb-autocomplete-grid-option` using the `option` property.
+
+```html
+<sbb-form-field label="Label">
+  <input />
+  <sbb-autocomplete-grid>
+    <sbb-autocomplete-grid-row>
+      <sbb-autocomplete-grid-option value="1">Option 1</sbb-autocomplete-grid-option>
+      <sbb-autocomplete-grid-cell>
+        <sbb-autocomplete-grid-button
+          id="button"
+          icon-name="pen-small"
+        ></sbb-autocomplete-grid-button>
+      </sbb-autocomplete-grid-cell>
+    </sbb-autocomplete-grid-row>
+  </sbb-autocomplete-grid>
+</sbb-form-field>
+
+<script>
+  const getOptionFromButton = (): SbbAutocompleteGridOptionElement => {
```

Comment:
Do you think showing how to get the option when receiving the event makes more sense?

Reply:
Function added